docs/hive-catalog-with-s3-and-adls.md

```diff
@@ -99,6 +117,7 @@ SupportsSchemas supportsSchemas = catalog.asSchemas();
 
 Map<String, String> schemaProperties = ImmutableMap.<String, String>builder()
     .put("location", "s3a://bucket-name/path")
+    // .put("location", "abfss://container-name@user-account-name.dfs.core.windows.net/path")
```

Comment:
You'd better add a comment here to say this is for ADLS.

Reply:
added.